symengine/ruby/ext/symengine/symengine.c

```diff
@@ -22,4 +24,11 @@ void Init_symengine() {
     rb_define_method(c_basic, "/", cbasic_div, 1);
     rb_define_method(c_basic, "**", cbasic_pow, 1);
     rb_define_method(c_basic, "-@", cbasic_neg, 0);
+    rb_define_method(c_basic, "to_s", cbasic_to_str, 0);
+    rb_define_method(c_basic, "expand", cbasic_expand, 0);
+
+    //Symbol class
+    c_symbol = rb_define_class_under(m_symengine, "Symbol", c_basic);
+    rb_define_alloc_func(c_symbol, csymbol_alloc);
```

Comment:
Yes, it is. Both are the same, as you noticed. This was just for clarity in code. We will have the freedom to modify what `csymbol_alloc` does, without having to change anything in this file in the future.